phpBB/phpbb/datetime.php

```diff
@@ -101,15 +101,15 @@ public function format($format = '', $force_absolute = false)
 				{
```

Comment:
shouldn't this be `if ($timestamp < $midnight + 2 * 86400)`

Reply:
Good catch.